unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -437,13 +439,16 @@ public override void RotateRight(ServerAction action)
 
             if (CheckIfAgentCanRotate("right", action.degrees)||action.forceAction) 
             {
+
+                base.RotateRight(action);
+
                 //only default hand if not manually Interacting with things
                 if(!action.manualInteract)
                 {
                     DefaultAgentHand();
                 }
 
-                base.RotateRight(action);
+                return;
```

Comment:
Can you remove these `return` statements? It doesn't look like anything comes after them.